tilequeue/query.py

```diff
@@ -150,15 +150,15 @@ def enqueue_queries(sql_conns, thread_pool, layer_data, zoom, unpadded_bounds):
 
 class DataFetcher(object):
 
-    def __init__(self, conn_info, layer_data, io_pool, n_conn, max_conn):
+    def __init__(self, conn_info, layer_data, io_pool, n_conn):
         self.conn_info = dict(conn_info)
         self.layer_data = layer_data
         self.io_pool = io_pool
 
         self.dbnames = self.conn_info.pop('dbnames')
         self.dbnames_query_index = 0
-        self.sql_conn_pool = DatabaseCycleConnectionPool(
-            n_conn, max_conn, self.dbnames, self.conn_info)
+        self.sql_conn_pool = DBAffinityConnectionsNoLimit(
+            self.dbnames, self.conn_info)
```

Comment:
This section around line 160 used to read:

```
        self.sql_conn_pool = DBAffinityConnectionsNoLimit(
            self.dbnames, n_conn, self.conn_info)
```

Where `self.n_conn = n_conn` was part of the function arguments. 

Similarly on 170 we used to say:


```
        sql_conns, conn_info = self.sql_conn_pool.get_conns()
``` 

instead of proposed: 

```
         sql_conns = self.sql_conn_pool.get_conns(self.n_conn)
```